sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/QueryBuilders/QueryBuilderSamples.cs

```diff
@@ -0,0 +1,163 @@
+            // SELECT Temperature FROM DigitalTwins T
+            new DigitalTwinsQueryBuilder<ConferenceRoom>()
+                .Select(r => r.Temperature)
+                .From(DigitalTwinsCollection.DigitalTwins, "T")
+                .Build();
+
+            // SELECT Temperature FROM DigitalTwins
+            new DigitalTwinsQueryBuilder<ConferenceRoom>()
+                .Select(r => r.Temperature)
+                .FromCustom("DigitalTwins")
+                .Build();
+            #endregion
+
+            // SELECT Room, Temperature From DIGTIALTWINS
+            DigitalTwinsQueryBuilder<ConferenceRoom> queryWithMultipleProperties = new DigitalTwinsQueryBuilder<ConferenceRoom>()
+                .Select("Room", "Temperature");
+
+            // SELECT * FROM DIGITALTWINS WHERE TEMPERATURE < 5
+            DigitalTwinsQueryBuilder<ConferenceRoom> queryWithComparisonWhereClause = new DigitalTwinsQueryBuilder<ConferenceRoom>()
+                .Where(r => r.Temperature < 5);
+
+            // SELECT * FROM DIGITALTWINS WHERE IS_OF_MODEL('dtmi:example:room;1', exact)
+            DigitalTwinsQueryBuilder<ConferenceRoom> queryWithIsOfModelExact = new DigitalTwinsQueryBuilder<ConferenceRoom>()
+                .Where(_ => DigitalTwinsFunctions.IsOfModel("dtmi:example:room;1", true));
+
+            // SELECT * FROM DIGITALTWINS WHERE Temperature = 50 AND IS_OF_MODEL("dtmi..", exact)
+            DigitalTwinsQueryBuilder<ConferenceRoom> logicalOps_SingleAnd = new DigitalTwinsQueryBuilder<ConferenceRoom>()
+                .Where(r => r.Temperature == 50 && DigitalTwinsFunctions.IsOfModel("dtmi:example:room;1", true));
+
+            // SELECT * FROM DIGITALTWINS WHERE Temperature = 50 OR IS_OF_MODEL("dtmi..", exact)
+            DigitalTwinsQueryBuilder<ConferenceRoom> logicalOps_SingleOr = new DigitalTwinsQueryBuilder<ConferenceRoom>()
+                .Where(r => (r.Temperature == 50 || DigitalTwinsFunctions.IsOfModel("dtmi:example:room;1", true)) && r.IsOccupied == true);
+
+            #region Snippet:DigitalTwinsQueryBuilder_ComplexConditions
+            // SELECT * FROM DIGITALTWINS WHERE Temperature = 50 OR IS_OF_MODEL("dtmi..", exact) OR IS_NUMBER(Temperature)
+            DigitalTwinsQueryBuilder<ConferenceRoom> logicalOps_MultipleOr = new DigitalTwinsQueryBuilder<ConferenceRoom>()
+                .Where(r => r.Temperature == 50 || 
+                DigitalTwinsFunctions.IsOfModel("dtmi:example:room;1", true) || 
+                DigitalTwinsFunctions.IsNumber(r.Temperature));
+
+            // SELECT * FROM DIGITALTWINS WHERE (IS_NUMBER(Humidity) OR IS_DEFINED(Humidity)) 
+            // OR (IS_OF_MODEL("dtmi:example:hvac;1") AND IS_NULL(Occupants))
+            DigitalTwinsQueryBuilder<ConferenceRoom> logicalOpsNested = new DigitalTwinsQueryBuilder<ConferenceRoom>()
+                .Where(r => (DigitalTwinsFunctions.IsNumber(r.Humidity) 
+                                || DigitalTwinsFunctions.IsDefined(r.Humidity))
```

Comment:
the indentation needs some love.

Reply:
![image](https://user-images.githubusercontent.com/84345221/127693962-b95ec0e4-2c5e-4c76-aaa5-bae903d690dd.png)
thoughts on this?